elastic/kibana
# [Serverless] Add lower size limit to Serverless elasticsearch query rule

Resolves https://github.com/elastic/response-ops-team/issues/154

## Summary

This PR updates the max size to 100 for the ES Query rule and sets the default size to 10.


### Checklist



### To verify

1. Run serverless
```
yarn es serverless --projectType oblt

yarn serverless-oblt
```
3. Create an ES Query rule in the UI and verify that the default value for `size` is 10, and you can't save a rule with `size` > 100
4. Go to [dev tools](http://localhost:5601/app/dev_tools#/console) and create a rule, verify that you get an error if the `size` > 100
```
POST kbn:/api/alerting/rule
{
  "params": {
    "searchType": "esQuery",
    "timeWindowSize": 5,
    "timeWindowUnit": "m",
    "threshold": [
      1000
    ],
    "thresholdComparator": ">",
    "size": 1000,
    "esQuery": "{\n    \"query\":{\n      \"match_all\" : {}\n    }\n  }",
    "aggType": "count",
    "groupBy": "all",
    "termSize": 5,
    "excludeHitsFromPreviousRun": false,
    "sourceFields": [],
    "index": [
      ".kibana"
    ],
    "timeField": "created_at"
  },
  "consumer": "stackAlerts",
  "schedule": {
    "interval": "1m"
  },
  "tags": [
    "serverless"
  ],
  "name": "test",
  "rule_type_id": ".es-query",
  "actions": []
}
```
